CHANGELOG.md

```diff
@@ -0,0 +1,73 @@
+- Solving Some LINQ Predicates issues [`4ed9782`](https://github.com/commercetools/commercetools-dotnet-core-sdk/commit/4ed978208dadedf828f165239bc1703e8514ecae)
+- Fixing property names in some models like:
+    - property `FieldName` in Type/UpdateActions/ChangeEnumValueOrderUpdateAction
+    - property `FieldName` in Type/UpdateActions/ChangeLocalizedEnumValueOrderUpdateAction
+- Adding missing properties in some models like: property `Key` in CartDiscount and CartDiscountDraft Models
+
+### Breaking Changes
+
+##### Zones:
+- Changing type of property `Name` in UpdateActions/ChangeNameUpdateAction from `LocalizedString` to `string`
+
+##### ProductTypes:
+- Changing namespace of all Update Actions of ProductTypes from commercetools.Sdk.Domain.ProductTypes to commercetools.Sdk.Domain.ProductTypes.UpdateActions
+- Changing type of property `Name` in ProductTypes/UpdateActions/ChangeNameUpdateAction from `LocalizedString` to `string`
+- Changing type of property `Description` in ProductTypes/UpdateActions/ChangeDescriptionUpdateAction from `LocalizedString` to `string`
+
+##### TaxCategories:
+- Changing namespace of all models of TaxCategories from commercetools.Sdk.Domain to commercetools.Sdk.Domain.TaxCategories
+- Changing namespace of all update Actions of TaxCategories from commercetools.Sdk.Domain.TaxRates to commercetools.Sdk.Domain.TaxCategories.UpdateActions
+
+##### CartDiscounts:
+- Changing namespace of all models of CartDiscounts from commercetools.Sdk.Domain to commercetools.Sdk.Domain.CartDiscounts
+- Changing namespace of all update Actions of CartDiscounts from commercetools.Sdk.Domain.CartDiscounts to commercetools.Sdk.Domain.CartDiscounts.UpdateActions
+
+##### DiscountCodes:
+- Changing namespace of all models of DiscountCodes from commercetools.Sdk.Domain to commercetools.Sdk.Domain.DiscountCodes
+- Changing namespace of all update Actions of DiscountCodes from commercetools.Sdk.Domain.DiscountCodes to commercetools.Sdk.Domain.DiscountCodes.UpdateActions
+       
+##### ShippingMethods
+- Changing Type of property `ShippingRates` to `List<ShippingRateDraft>` instead of `List<ShippingRate>` in ZoneRateDraft
+- Changing type of property `Name` in UpdateActions/ChangeNameUpdateAction from `LocalizedString` to `string`
+
+##### Replace properties of type `CustomFields` with type `CustomFieldsDraft` in these Draft Models 
+- ShoppingLists/ShoppingListDraft
+- ShoppingLists/TextLineItemDraft
+- Carts/CartDraft
+- Carts/CustomLineItemDraft 
+- Carts/LineItemDraft
+- Carts/UpdateActions/AddLineItemUpdateAction
+
+##### Solving type of properties in some domain models,[`3beb7eda`](https://github.com/commercetools/commercetools-dotnet-core-sdk/commit/3beb7edaa8cfc938d48b12efcbc1f62bd56f1ddd)
+##### Changing namespace of Delivery model from commercetools.Sdk.Domain to commercetools.Sdk.Domain.Carts 
+##### Convert property `ProductId` in GetMatchingProductDiscountParameters model to be `string` instead of `Guid`
```

Comment:
```suggestion
##### ProductDiscounts
- Convert property `ProductId` in GetMatchingProductDiscountParameters model to be `string` instead of `Guid`
```